.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -38,7 +38,7 @@ body:
       label: Host OS system
       placeholder: |
         Example:
-        Ubuntu 22.04
+        Ubuntu 24.04
```

Comment:
This is unrelated, but we can update it because we expect more users to start ubuntu 24.04 as their host system as well